src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ModelRunTests.kt

```diff
@@ -46,4 +47,26 @@ class ModelRunTests : SecureIntegrationTests() {
         val responseEntity = testRestTemplate.getForEntity<String>("/model/status/${id}")
         assertSecureWithSuccess(isAuthorized, responseEntity, "ModelStatusResponse")
     }
+
+    @ParameterizedTest
+    @EnumSource(IsAuthorized::class)
+    fun `can get model run result`(isAuthorized: IsAuthorized) {
+
+        val id = when(isAuthorized){
+            IsAuthorized.TRUE -> {
+                val entity = getJsonEntity()
+                val runResult = testRestTemplate.postForEntity<String>("/model/run/", entity)
+                ObjectMapper().readValue<JsonNode>(runResult.body!!)["data"]["id"].textValue()
+            }
+            IsAuthorized.FALSE -> "nonsense"
+        }
+
+        do {
```

Comment:
This may stop being practical as a test once we have real model running.. might need some sort of test mode?